src/tests/unit/cpu/mkldnn_jit_kernel_test.cpp

```diff
@@ -0,0 +1,347 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
please remove `mkldnn_` from the name of the file.

Reply:
Test file was renamed.